tests/sles4sap/hana_cluster.pm

```diff
@@ -53,7 +53,7 @@ sub run {
 
     if (is_node(1)) {
         # Create the resource configuration
```

Comment:
Are there any places in the code where we documented what we have special in each of the 3 .conf files?
If not maybe we can put it here expanding `# Create the resource configuration` to briefly describe each of them.

Reply:
Add the more comments here. Thanks